header.php

```diff
@@ -19,6 +19,7 @@
 	<meta name="theme" content="MOJ-hale-1.0.11">
 	<meta name="description" content="<?php bloginfo('description'); ?>" />
 	<link rel="profile" href="https://gmpg.org/xfn/11">
+  <link rel="stylesheet" id="custom-colour-css" href="https://jotwpublic.docker/wp-includes/css/MjB-test.css?time=<?php echo $_SERVER['REQUEST_TIME']; ?>" media="all">
```

Comment:
Referencing the PHP-generated CSS file like this in the header.  Is this okay?

Reply:
I will look at moving this to s3 bucket/uploads folder. But happy to discuss.